lighthouse-core/audits/offline-start-url.js

```diff
@@ -56,6 +56,19 @@ class OfflineStartUrl extends Audit {
 
     const hasOfflineStartUrl = artifacts.StartUrl.statusCode === 200;
 
+    // The StartUrl artifact provides no explanation if a response was received from a service
+    // worker with a non-200 status code. In that case, this audit provides its own explanation.
+    // In all other cases it defers to the artifact explanation.
+    const explanation = artifacts.StartUrl.explanation;
+    // if (!explanation && artifacts.StartUrl.statusCode !== -1 && !hasOfflineStartUrl) {
```

Comment:
presumably you're gonna nuke these lines?

Reply:
based on feedback re: warning or explanation, yes

Reply:
explanation makes more sense IMO